src/pickley/__init__.py

```diff
@@ -1038,3 +1069,26 @@ def represented(self):
 
         result.append("")
         return "\n".join(result)
+
+
+def latest_pypi_version(package_name, index):
+    package_name = PypiStd.std_package_name(package_name)
+    if package_name:
+        url = f"{index.rstrip('/')}/{package_name}/"
+        response = http_get(url)
+        if response:
+            versions = sorted(i.version for i in _parsed_simple_html(response) if i.version.is_final)
```

Comment:
`max` with a try/except might be simpler?

Reply:
I could change `sorted()` to `max()` indeed here, no need for try/except (as only valid versions are yielded)

Reply:
Ah I see what you mean, for an empty `/simple/` response. I'll do it because it adds one more test edge case :)